### PR TITLE
Phg 116

### PR DESCRIPTION
**Pull request by richardli515**

### CODE REVIEW

File: src/components/search/SearchResult.vue [hunk 1]
```diff
@@ -106,23 +102,24 @@
...
         methods: {
             ...mapActions({
                 stateTreePaginate: types.TREE_ACTION_PAGINATE,
                 setFilter: types.TREE_ACTION_SET_FILTER
             }),
-            newSearch() {
-              this.gotoPage(1);
-            },
-            gotoPage(page) {
+            // newSearch() {
```
Comment (richardli515) [inline src/components/search/SearchResult.vue]: delete the funciton

File: src/views/TreeBrowse.vue [hunk 2]
```diff
@@ -92,9 +94,9 @@
                     filters: this.stateTreeFilters
                 }
                 this.stateAction_doSearch(payload);
-                if(this.$refs.sr) {
-                    this.$refs.sr.newSearch();
-                }
+                // if(this.$refs.sr) {
```
Comment (richardli515) [inline src/views/TreeBrowse.vue]: check if we can remove $refs.sr and $ refs.sf

File: src/views/TreeBrowse.vue [hunk 3]
```diff
@@ -68,13 +68,15 @@
                 treeFilters: null
             }
         },
-        beforeRouteEnter (to, from, next) {
-            next(vm => {
-                vm.doSearch();
-            });
-        },
+        // beforeRouteEnter (to, from, next) {
```
Comment (trilok2000) [inline src/views/TreeBrowse.vue]: can we remove this commented code?

File: src/views/TreeBrowse.vue [hunk 4]
```diff
@@ -68,13 +68,15 @@
...
+        //     next(vm => {
+        //         vm.doSearch();
+        //     });
+        // },
```
Comment (trilok2000) [inline src/views/TreeBrowse.vue]: can we remove this commented code?

File: src/views/TreeBrowse.vue [hunk 5]
```diff
@@ -120,11 +122,11 @@
...
-            stateSearchText: {
-                handler: function (val, oldVal) {
-                    this.doSearch();
-                }
-            },
+            // stateSearchText: {
+            //     handler: function (val, oldVal) {
+            //         this.doSearch();
+            //     }
+            // },
```
Comment (trilok2000) [inline src/views/TreeBrowse.vue]: can we remove this commented code?

File: src/components/search/SearchResult.vue [hunk 6]
```diff
@@ -106,23 +102,24 @@
...
-                    return;
+            //     if(page === 0 || page > this.noPages || this.currentPage === page)
+            //         return;
 
-                this.treeFilters.startRow = (page - 1) * this.treeFilters.rows;
-                this.stateTreePaginate(this.treeFilters);
-            },
+            //     this.treeFilters.startRow = (page - 1) * this.treeFilters.rows;
+            //     this.stateTreePaginate(this.treeFilters);
+            // },
```
Comment (trilok2000) [inline src/components/search/SearchResult.vue]: can we remove this commented code?

File: src/components/search/SearchResult.vue [hunk 7]
```diff
@@ -106,23 +102,24 @@
...
-            gotoPage(page) {
+            // newSearch() {
+            //   this.gotoPage(this.currentPage);
+            // },
+            // gotoPage(page) {
 
-                if(page === 0 || page > this.noPages || this.currentPage === page)
-                    return;
+            //     if(page === 0 || page > this.noPages || this.currentPage === page)
+            //         return;
```
Comment (trilok2000) [inline src/components/search/SearchResult.vue]: can we remove this commented code?

File: src/components/search/SearchResult.vue [hunk 8]
```diff
@@ -106,23 +102,24 @@
...
                 setFilter: types.TREE_ACTION_SET_FILTER
             }),
-            newSearch() {
-              this.gotoPage(1);
-            },
-            gotoPage(page) {
+            // newSearch() {
+            //   this.gotoPage(this.currentPage);
+            // },
+            // gotoPage(page) {
```
Comment (trilok2000) [inline src/components/search/SearchResult.vue]: can we remove this commented code?

**Review by trilok2000 — CHANGES_REQUESTED** [review]
pls remove the commented code. and i think we are good to go with merge. thanks!

File: src/views/TreeBrowse.vue (hunk 3 above)
Comment (richardli515) [reply src/views/TreeBrowse.vue]: yes. removed

File: src/views/TreeBrowse.vue (hunk 4 above)
Comment (richardli515) [reply src/views/TreeBrowse.vue]: yes. removed.

File: src/views/TreeBrowse.vue (hunk 5 above)
Comment (richardli515) [reply src/views/TreeBrowse.vue]: yes. removed.

File: src/components/search/SearchResult.vue (hunk 8 above)
Comment (richardli515) [reply src/components/search/SearchResult.vue]: yes. removed.

File: src/components/search/SearchResult.vue (hunk 7 above)
Comment (richardli515) [reply src/components/search/SearchResult.vue]: removed

File: src/components/search/SearchResult.vue (hunk 6 above)
Comment (richardli515) [reply src/components/search/SearchResult.vue]: removed

File: src/components/search/SearchResult.vue (hunk 8 above)
Comment (richardli515) [reply src/components/search/SearchResult.vue]: removed

**Pull request merged**